geometry_msgs/Point position
---
bool received